src/analyzer_main.cpp

```diff
@@ -559,6 +560,58 @@ struct Analyzer {
             return false;
         }
 
+        //Handling susceptibility
+        if(config.use_susceptibility == true && stats.n_steps % 10 == 0){
+
+            ofstream myfile;
+
+            stringstream ss;
+            ss << "output/change_in_ideology_step_" << stats.n_steps << ".dat";
+            string filename = ss.str();
+            myfile.open (filename.c_str());
+
+            myfile << "\nn_steps: " << stats.n_steps;
+
+            int old_ideology;
+
+            for (Agent& agent : network) {
+                if (agent.susceptibility == 1.0) {
+
+                    myfile << "\n\nagent_id: " << agent.id;
+
+                    old_ideology = agent.ideology_bin;                
+
+                    // Looking at the ideology of all of an agent's followings:
+                    int counts[N_BIN_IDEOLOGIES];
+                    for (int& count : counts) {
+                        count = 0; 
+                        // Counts start at 0
+                    }
+                    for (int following_id : agent.following_set.as_vector()) {
+                        Agent& following = network[following_id];
+                        counts[following.ideology_bin]++;
+                    }
+
+                    int most_common_ideology = -1;
+                    int max_count = -1;
+                    for(int i=0; i<N_BIN_IDEOLOGIES; i++){
```

Comment:
Nit: Spacing again. You can get Eclipse to autoformat the spacing with ctrl+shift+f, there should be space like `i < N_BIN_IDEOLOGIES` and `for (int`.